po/da.po

```diff
@@ -14915,51 +14918,68 @@ msgid ""
 "Sometimes includes small exercises where you are asked to expand on the code "
 "in the examples."
 msgstr ""
+"[Rust By Example](https://doc.rust-lang.org/rust-by-example/): dækker "
+"syntaksen i Rust via en række eksempler som viser de forskellige "
+"konstruktioner. Der er sommetider små opgaver hvor du bliver bedt om at "
+"udvide koden i eksemplerne."
 
 #: src/other-resources.md:17
 msgid ""
 "[Rust Standard Library](https://doc.rust-lang.org/std/): full documentation "
 "of the standard library for Rust."
 msgstr ""
+"[Rust Standard Library](https://doc.rust-lang.org/std/): fuld dokumentation "
+"af standardbiblioteket for Rust."
 
 #: src/other-resources.md:19
 msgid ""
 "[The Rust Reference](https://doc.rust-lang.org/reference/): an incomplete "
 "book which describes the Rust grammar and memory model."
 msgstr ""
+"[The Rust Reference](https://doc.rust-lang.org/reference/): en ufuldstændig "
+"bog som beskriver grammatikken og en hukommelsesmodel for Rust."
 
 #: src/other-resources.md:22
 msgid "More specialized guides hosted on the official Rust site:"
-msgstr ""
+msgstr "Mere specialicerede guider hostet på den officielle Rust side:"
```

Comment:
nit: "Mere specialiserede guider huset på den officielle Rust side:"? 

I think "specialicerede" should be "specialiserede".

Reply:
Thanks!

```suggestion
msgstr "Mere specialiserede guider huset på den officielle Rust side:"
```